Doc/library/doctest.rst

```diff
@@ -584,20 +583,7 @@ doctest decides whether actual output matches an example's expected output:
    Note that :const:`ELLIPSIS` can also be used to ignore the
    details of the exception message, but such a test may still fail based
    on whether or not the module details are printed as part of the
-   exception name. Using :const:`IGNORE_EXCEPTION_DETAIL` and the details
```

Comment:
It was like this before your PR, but I don't really understand what this sentence means.